tools/shared.py

```diff
@@ -2979,8 +2979,20 @@ def delebify(buf, offset):
 
   @staticmethod
   def add_emscripten_metadata(js_file, wasm_file):
-    mem_size = Settings.STATIC_BUMP
+    WASM_PAGE_SIZE = 65536
+
+    mem_size = Settings.TOTAL_MEMORY // WASM_PAGE_SIZE
```

Comment:
Doesn't `//` so the wrong thing here and round down?    I guess TOTAL_MEMORY is already guaranteed to be a multiple so probably fine.

Reply:
Yeah, the intent is to make sure mem_size is an integer. I could have done `int(TOTAL_MEMORY / WASM_PAGE_SIZE)` but this seemed better.